boot/makebootable_test.go

```diff
@@ -503,20 +559,18 @@ func (s *makeBootable20UbootSuite) TestUbootMakeBootable20RunMode(c *C) {
 	c.Assert(err, IsNil)
 
 	// uboot on ubuntu-seed
-	mockSeedUbootEnv := filepath.Join(boot.InitramfsUbuntuSeedDir, "uboot.env")
-	err = os.MkdirAll(filepath.Dir(mockSeedUbootEnv), 0755)
+	mockSeedUbootBootSel := filepath.Join(boot.InitramfsUbuntuSeedDir, "uboot/ubuntu/boot.sel")
```

Comment:
we need some TODO comment in the relevant code about trying to avoid to need to have the file in place already, but consider the seed bootloader to pick what to do with the boot partition

Reply:
we have it in the uboot file itself, do you also want a TODO in the tests?

Reply:
I think we want a TODO comment in makeBootable20RunMode ?  it's kind of an issue across bootloaders, no?

Reply:
ah yes of course, sure